docs/HowToUsePeripheral.md

```diff
@@ -423,18 +437,18 @@ Please refer to **[i2c.h](../os/include/tinyara/i2c.h)** to know more about the
 
 ## I2S
 
-I2S port for a board involves the following;
-1. I2S Device Structure and its Operations.
+I2S port for a board involves the following;  
+1. I2S Device Structure and its Operations.  
 2. I2S Initialization and Device Access.  
 
 ### I2S Device Structure and Operations
 Each board, shall;  
 
-Implement and expose an instance of struct i2s_dev_s for each port.     
-Implement the supported operations on i2s, ie struct i2s_op_s. 
-Data structures below; 
+Implement and expose an instance of struct i2s_dev_s for each port.  
+Implement the supported operations on i2s, ie struct i2s_op_s.  
+Data structures below;  
```

Comment:
Please remove trailing space if possible.

Reply:
the two spaces mean 'change lines', It is one of markdown syntax.